File: Examples/test-suite/ruby/typedef_scope_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# Author::    gga
# Copyright:: 2007
# License::   SWIG
#

require 'swig_assert'

require 'typedef_scope'

b = Typedef_scope::Bar.new

x = b.test1(42, "hello")
if x != 42
  puts "Failed!!"
end

x = b.test2(42, "hello")
if x != "hello"
  puts "Failed!!"
end

File: Examples/test-suite/perl5/typedef_scope_runme.pl
use strict;
use warnings;
use Test::More tests => 4;
BEGIN { use_ok('typedef_scope') }
require_ok('typedef_scope');

my $b = typedef_scope::Bar->new();
my $x = $b->test1(42, "hello");
is($x, 42, "Failed!!");
$x = $b->test2(42, "hello");
is($x, "hello", "Failed!!");
    
    
